capture-traffic/capture-from-remote-pc.md

```diff
@@ -0,0 +1,93 @@
+
+
+## Capturing Remote Windows Traffic
+
+Fiddler Everywhere provides an automated guide to configure remote Windows capturing through the following steps:
+
+1. Start Fiddler Everywhere. 
+
+1. Open the **Home** pane.
+
+1. Open the **Remote Devices** screen.
+
+1. Open the **Windows** tutorial and follow the instructions.
+
+The tutorial requires administrative privileges for the current user (on the remote host) to install Fiddler's CA (certificate authority) and to set the system proxy. Once the steps are executed, you can immediately capture traffic from the remote Windows device.
+
+
+## Capturing Remote macOS Traffic
+
+Fiddler Everywhere provides an automated guide to configure remote macOS capturing through the following steps:
+
+1. Start Fiddler Everywhere. 
+
+1. Open the **Home** pane.
+
+1. Open the **Remote Devices** screen.
+
+1. Open the **macOS** tutorial and follow the instructions.
+
+The tutorial requires administrative privileges for the current user (on the remote host) to install Fiddler's CA (certificate authority) and to set the system proxy. Once the steps are executed, you can immediately capture traffic from the remote macOS device.
+
+
+## Configuring Other Generic Devices (Manual Setup)
+
+You can manually configure both the Fiddler Everywhere and the remote device. Notice that the instructions below apply only to remote devices that allow configuring manual proxy and installing the Fiddler CA (certificate authority).
+
+1. [Prepare Fiddler Everywhere host for capturing remote traffic](#configuring-the-fiddler-everywhere-host).
+
+1. [Configure the Fiddler Everywhere proxy on remote device](#configuring-the-fiddler-proxy-on-the-remote-device).
+
+
+### Configuring the Fiddler Everywhere Host
+
+1. Enable the remote connections of Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**. Combine it with **Keep it ON after app restart** if you want to persist the setting for subsequent application startups.
+
+1. Check the local IP address of the Fiddler Everywhere application host. You can use [the connection status on the lower right-hand side]({%slug connections-section%}) to get the host IP address. Alternatively, you can get the host IP address using a terminal command like **ipconfig**/ **ifconfig** (depending on the OS).
+
+    ![Host local IP address](../images/kb/host-ip.png)
+
+
+### Configuring the Fiddler proxy on the Remote Device
+
+1. Go to the operating system proxy settings on the target machine.
```

Comment:
The steps for setting the proxy must be last - after successful trust of the root certificate. This is the way we present them in the wizards and I suggest following the same approach here.